src/main/resources/META-INF/common_at.cfg

```diff
@@ -0,0 +1,4 @@
+# Accessors have made great strides in the reduction of access transformers, from almost 100+ to one... Sadly it's enums which beat us up again.
```

Comment:
No need for the extra comment, can just specify that AT's are heavily discouraged unless they can't be feasibly replaced with an Accessor mixin.